backend/api/Controllers/ReturnToHomeController.cs

```diff
@@ -0,0 +1,61 @@
+﻿using Api.Controllers.Models;
+using Api.Database.Models;
+using Api.Services;
+using Microsoft.AspNetCore.Authorization;
+using Microsoft.AspNetCore.Mvc;
+namespace Api.Controllers
+{
+    [ApiController]
+    [Route("return-to-home")]
+    public class ReturnToHomeController(
+            ILogger<RobotController> logger,
+            IReturnToHomeService returnToHomeService,
+            IRobotService robotService
+        ) : ControllerBase
+    {
+        /// <summary>
+        ///     Sends the robots to their home.
+        /// </summary>
+        [HttpPost("schedule-return-to-home/{robotId}")]
+        [Authorize(Roles = Role.User)]
+        [ProducesResponseType(typeof(MissionRun), StatusCodes.Status201Created)]
+        [ProducesResponseType(StatusCodes.Status400BadRequest)]
+        [ProducesResponseType(StatusCodes.Status401Unauthorized)]
+        [ProducesResponseType(StatusCodes.Status403Forbidden)]
+        [ProducesResponseType(StatusCodes.Status404NotFound)]
+        [ProducesResponseType(StatusCodes.Status500InternalServerError)]
+        public async Task<ActionResult<IList<MissionRun>>> ScheduleReturnToHomeMission(
+            [FromRoute] string robotId
+        )
+        {
+            var robot = await robotService.ReadById(robotId);
+            if (robot is null)
+            {
+                logger.LogWarning("Could not find robot with id={Id}", robotId);
+                return NotFound();
+            }
+
+            if (!robot.IsRobotPressureHighEnoughToStartMission())
+            {
+                return BadRequest($"The robot pressure on {robot.Name} is too low to start a mission");
+            }
+
+            if (!robot.IsRobotBatteryLevelHighEnoughToStartMissions())
+            {
+                return BadRequest($"The robot battery level on {robot.Name} is too low to start a mission");
+            }
```

Comment:
I'm not sure, probably yes. For taurob at least, if the pressure is too low I think the robot will turn off immediatly if in atex mode. ExR I'm not certain of what happens... for the battery case it should definetly be ignored because if you have low battery you actually want to go back to the robot home asap. But then remove the battery one and what about the pressure?